plugins/SliceInfoPlugin/example_data.json

```diff
@@ -0,0 +1,101 @@
+{
+    "extruders": [
```

Comment:
Code style: Brackets on new line. Same goes for the rest in this file.

Reply:
The updated example is formatted with a JSON formatter.